--- lib/hostname.cc	Mon Mar  4 23:03:42 2002
+++ lib/hostname.cc	Mon Mar  4 23:04:07 2002
@@ -21,6 +21,7 @@
 
 #include "config.h"
 #include "mystring/mystring.h"
+#include <string.h>
 #include <unistd.h>
 #include <sys/utsname.h>
 
--- src/send.cc	Mon Mar  4 23:03:47 2002
+++ src/send.cc	Mon Mar  4 23:04:23 2002
@@ -30,6 +30,7 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <unistd.h>
+#include <stdlib.h>
 #include "configio.h"
 #include "defines.h"
 #include "errcodes.h"
