--- smtp.cc     2000/12/30 03:41:21     1.5
+++ smtp.cc     2001/06/21 20:50:19
@@ -120,9 +120,8 @@
   docmd("DATA", 300);
   mystring tmp;
   while(msg->getline(tmp)) {
-    if((tmp[0] =3D=3D '.' && tmp[1] =3D=3D 0 && !(out << ".")) ||
-       !(out << tmp << "\r\n"))
-      exit(ERR_MSG_WRITE);
+    if(tmp[0] =3D=3D '.' && !(out << ".")) exit(ERR_MSG_WRITE);
+    if(!(out << tmp << "\r\n")) exit(ERR_MSG_WRITE);
   }
   docmd(".", 200);
 }
