From 8439727803322c055d20f3a2b461131bdc09026a Mon Sep 17 00:00:00 2001
From: Nick Leverton <nick@leverton.org>
Date: Tue, 11 Jul 2017 22:24:54 -0300
Subject: [PATCH 3/5] 12_allow_no_dots.diff

---
 lib/hostname.cc | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/lib/hostname.cc b/lib/hostname.cc
index c0b2832..a2845e9 100644
--- a/lib/hostname.cc
+++ b/lib/hostname.cc
@@ -33,12 +33,16 @@ void read_hostnames()
 {
   int nome;
   nome = 0;
-  if (!config_read("me", me)) {
+  // introduced as bufix for #120660, #157259, #158412 in 1.00RC5-17;
+  // still there since it's more appropriate for Debian systems
+  if (!config_read("../mailname", me)) {
     nome = 1;
     me = "me";
   }
-  if (!config_read("defaultdomain", defaultdomain))
-    defaultdomain = nome ? "defaultdomain" : me.c_str();
+  // allow defaultdomain to be empty for Debian
+  config_read("defaultdomain", defaultdomain);
+  // if (!config_read("defaultdomain", defaultdomain))
+    // defaultdomain = nome ? "defaultdomain" : me.c_str();
   if (!config_read("defaulthost", defaulthost))
     defaulthost = nome ? "defaulthost" : me.c_str();
   canonicalize(defaulthost);
-- 
2.13.2

